modules/osdk-ensuring-operator-workloads-run-restricted-psa.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-ensuring-operator-workloads-run-restricted-psa_{context}"]
+= Ensuring Operator workloads run in namespaces set to the restricted pod security level
+
+If you ensure your Operator workloads can run in namespaces set to the `restricted` pod security level, your Operator project can run on a wide variety of deployments and environments.
```

Comment:
This sentence sounds odd. Maybe lead with the want (Operator project can run...) before the how? 

>To ensure your Operator project can run on a wide variety of deployments and environments, configure the  Operator workloads to run in namespaces set to the `restricted` pod security level.

Reply:
Great suggestion. Thank you.